wwwroot/buy-anything.html

```diff
@@ -0,0 +1,414 @@
+                        <div class="col-12">
+                            <div class="card">
+                                <input type="checkbox" id="faq1">
+                                <label for="faq1">
+                                    1. Can I really buy anything?
+                                </label>
+                                <div class="card-content">
+                                    <p>Shopinbit's Concierge Service does not provide any kind of drugs, prescription medications, weapons, precious metals, cryptocurrencies, financial instruments, or any type of product that is illegal in Poland. You can buy anything else.</p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq2">
+                                <label for="faq2">
+                                    2. Can I pay my rent or another invoice?
+                                </label>
+                                <div class="card-content">
+                                    <p>Yes, Shopinbit can pay with almost any payment methods supported by either Wise, Revolut, the banking system and major credit cards, so services like your rent can be paid.</p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq3">
+                                <label for="faq3">
+                                    3. Why does it take 24-48 hours to get my quote?
+                                </label>
+                                <div class="card-content">
+                                    <p>All requests are handled manually by Shopinbit’s team and sometimes need to be reviewed with legal counsel. The following 4 points have to be reviewed:
+                                        <br>
+                                        <br> a) That the product/service can be bought from Shopinbit;
+                                        <br> b) Shopinbit is allowed to sell the product to you;
+                                        <br> c) Shopinbit can ship the product to you;
+                                        <br> d) It is all legal to do so.
+                                        <br>
+                                        <br> An offer will be issued only until these 4 points are sorted out. If it’s not possible, you will receive a friendly decline with an explanation of why.
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq4">
+                                <label for="faq4">
+                                    4. How is the bitcoin price calculated at the time of the transaction?
```

Comment:
Shouldn't it be "Bitcoin", not "bitcoin", in this context?
```suggestion
                                    4. How is the Bitcoin price calculated at the time of the transaction?
```